### PR TITLE
Fix voice over for add comment text in share extension

### PR DESCRIPTION
**Pull request by szabolcs-instructure**
refs: MBL-16316
affects: Student
release note: Fix voice over for add comment text in share extension

test plan: see ticket

## Checklist

- [x] Follow-up e2e test ticket created or not needed
- [x] A11y checked
- [x] Tested on phone
- [x] Tested on tablet
- [x] Approve from product or not needed

### CODE REVIEW

**Review by idevainstr — DISMISSED** [review]
QA +1 iOS 14.8.1, iPadOS 16.1b

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.67% | 91.67% | 0%
Core/Core/Submissions/SubmissionBreakdown/SubmissionBreakdownViewModelProtocol.swift | 0% | 0% | 0%
Core/Core/Quizzes/QuizDetails/ViewModel/PreviewQuizDetailsViewModel.swift | 0% | 0% | 0%

[MBL-16316](https://instructure.atlassian.net/browse/MBL-16316)

#### Affected Apps: Student

#### Release Note: 
Fix voice over for add comment text in share extension
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

**Review by vargaat — APPROVED** [review]
QA+1

File: Core/Core/SubmitAssignmentExtension/View/SubmitAssignmentExtensionView.swift [hunk 1]
```diff
@@ -80,7 +80,7 @@ public struct SubmitAssignmentExtensionView: View {
             .padding(.trailing, -20) // Offset parent's padding so our scrollbar will be in line with parent's scrollbar
             .padding(.leading, -5) // Offset TextEditor's default padding so we'll be in line with the course and assignment picker cells
             .overlay(placeholder, alignment: .topLeading)
-
+            .accessibilityLabel("Add optional comment")
```
Comment (vargaat) [inline Core/Core/SubmitAssignmentExtension/View/SubmitAssignmentExtensionView.swift:83]: This should be a localized string.

**Review by vargaat — CHANGES_REQUESTED** [review]

**Pull request merged**